cockroachdb/datadog_checks/cockroachdb/metrics.py

```diff
@@ -337,6 +337,7 @@
     'jobs_backup_resume_failed': 'jobs.backup.resume_failed',
     'jobs_backup_resume_retry_error': 'jobs.backup.resume_retry_error',
     'jobs_backup_resume_completed ': 'jobs.backup.resume_completed',
+    'schedules_BACKUP_last_completed_time': 'schedules.backup.last_completed_time',
```

Comment:
Can we also add this metric to the test suite?

Reply:
I believe this metric does get asserted in the integration and e2e tests: https://github.com/DataDog/integrations-core/blob/6728eee6d104daf2470645169e0263e38f12a2d0/cockroachdb/tests/common.py#L202

Reply:
Oh we directly import this map, my bad!